### DISCUSSION TITLE
Can we make the play command to not play songs from YT

### DISCUSSION THREAD

**Discussion by Huz-Devv** [SudhanPlayz/Discord-MusicBot#705] [Q&A] [answered]
Is there a way to stop the bot to not play from yt and only from soundcloud and etc

**Comment by DarrenOfficial** [comment, marked as answer]
Disable YouTube on your lavalink config.

**Comment by Huz-Devv** [comment]
> Disable YouTube on your lavalink config.

Well I did that but then when I play a Spotify playlist that don't work

Is there a way in which when we do play command it don't let the user play yt song and playlist but he can use other stuff like SoundCloud, Spotify etc
Like if "YouTube" in query and it don't plays

**Comment by DarrenOfficial** [comment]
Because the bot searches for spotify alternative on youtube

**Comment by DarrenOfficial** [comment]
it's impossible to play directly from spotify due to it's drm
